
{'date': 'Wed Mar 27 2019 20:16:12.904 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 9
    a1 = A()
TypeError: __init__() missing 1 positional argument: um_valor
'''},
{'date': 'Wed Mar 27 2019 20:25:57.23 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 16
    print(a0, a1, a0 == a1, isinstance(a0, A), a0.um_valor, a1.um_valor)
AttributeError: 'B' object has no attribute 'um_valor'
'''},
{'date': 'Wed Mar 27 2019 20:28:26.5 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 16
    print(a0, a1, a0 == a1, isinstance(a0, A), a0.um_valor, a1.um_valor)
AttributeError: 'B' object has no attribute 'um_valor'
'''},
{'date': 'Wed Mar 27 2019 20:30:19.288 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 16
    print(a0, a1, a0 == a1, isinstance(a0, A), a0.um_valor, a1.um_valor)
AttributeError: 'B' object has no attribute 'um_valor'
'''},
{'date': 'Wed Mar 27 2019 20:55:31.891 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''[<__main__.A object>, <__main__.A object>, <__main__.B object>, <__main__.B object>]
Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 26
    a.muda()
  module <module> line 11
    self.um_valor *= 2
TypeError: unsupported operand types for *: 'function' and 'int'
'''},
{'date': 'Wed Apr 03 2019 17:45:54.811 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''faz_algo 7
faz_qqc 7
Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 37
    faz_algo()
  module <module> line 33
    print("inspeciona o_lambda", dir(o_lambda), (o_lambda.__code__.co_locals))
AttributeError: 'code' object has no attribute 'co_locals'
'''},
{'date': 'Wed Apr 03 2019 17:46:26.834 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''faz_algo 7
faz_qqc 7
Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 37
    faz_algo()
  module <module> line 33
    print("inspeciona o_lambda", dir(o_lambda), (o_lambda.__code__).co_locals)
AttributeError: 'code' object has no attribute 'co_locals'
'''},